science/xschem/Portfile

```diff
@@ -0,0 +1,93 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem      1.0
+
+name            xschem
+version         3.4.6
+revision        0
+license         GPL-2
+categories      science cad
+
+maintainers     {gmail.com:degnan.68k @bpdegnan} \
+                openmaintainer
+
+description     schematic capture and netlisting EDA tool
+long_description \
+    Xschem is a schematic capture program that supports digital, analog, behavioral \
+    circuit design and simulation. It is designed to interface well with ngspice \
+    and other simulation tools.
+
+homepage        https://xschem.sourceforge.io/stefan/index.html
+
+distname        ${name}-${version}
+master_sites    master_sites    sourceforge:xschem
+
+
+checksums       rmd160  bf5f75f9e0838293515796bdead483ff2b667049 \
+                sha256  a46bd296b2b9f8436370892eb81264599c2e3ae84b298d7a4e3b89c4126185e7 \
+                size    14990039
+
+set docdir      ${prefix}/share/doc/${name}
+
+depends_build       port:pkgconfig
+depends_lib         port:cairo \
+                    port:xorg-libxcb \
+                    port:xrender \
+                    port:freetype \
+                    port:Xft2 \
+                    port:xpm \
+                    port:readline \
+                    port:tcl \
+                    port:tk \
+                    port:xorg-libX11
+                    
+use_configure   yes
+configure.args      --prefix=${prefix}
+
+destroot {
+    set bindir      ${destroot}${prefix}/bin
+    set sharedir    ${destroot}${prefix}/share/xschem
+    set docdir      ${destroot}${prefix}/share/doc/${name}
+
+    xinstall -d ${bindir} ${sharedir} ${docdir}
```

Comment:
I am pretty sure that MacPorts creates the `bin` and `share` directory already so that seems redundant, please double check